lib/asciidoctor/js/opal_ext/number.rb

```diff
@@ -0,0 +1,10 @@
+class Number < Numeric
+  def round(ndigits = undefined)
```

Comment:
@mojavelinux do you think we need a more sophisticated implementation ? As far as I know Asciidoctor core only uses this function to round the column/table size.

Reply:
That seems to do the trick. If all column width tests pass, I'd say we're good.

Reply:
> That seems to do the trick. If all column width tests pass, I'd say we're good.

I've added one, I will check the Asciidoctor tests suite to add more.